docs/api.rst

```diff
@@ -973,6 +973,152 @@ Query the historical price of assets
    :statuscode 500: Internal rotki error
    :statuscode 502: An external service used in the query such as cryptocompare/coingecko could not be reached or returned unexpected response.
 
+
+
+.. http:put:: /api/(version)/assets/prices/historical
+
+    Manually adds the price of an asset against another asset at a certain timestamp to the database.
+
+   .. note::
+      This endpoint can also be queried asynchronously by using ``"async_query": true``.
```

Comment:
Why would you make this call aynchronous? Isn't it just a DB query? For such calls we don't offer asynchronous option as they should be blazing fast and not need an async call.

Reply:
I thought async queries should be used with db always

Reply:
Nope. I don't understand what gave you this idea. Async queries are used for anything that may not be fast/almost instant. Which means ... pretty much anything that has to do with remote queries or really long calculations.

A local DB query is one of the few things that are fast. For example check the tag db queries.